Tasks/AzureAppConfigurationV10/task.json

```diff
@@ -0,0 +1,173 @@
+{
+    "id": "6b1a4059-eb7f-47b2-97e9-b959cb665b8d",
+    "name": "AzureAppConfiguration",
+    "friendlyName": "Azure App Configuration",
+    "description": "Download key-values to task variables from Azure App Configuration",
+    "helpUrl": "https://learn.microsoft.com/azure/azure-app-configuration/pull-key-value-devops-pipeline",
+    "helpMarkDown": "Email AzureAppConfig@microsoft.com for questions.",
+    "category": "Deploy",
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 10,
+        "Minor": 245,
```

Comment:
Why is the minor version starting out so high given this is a new major version?

Reply:
This is recommended in the docs-> [Bumping task version](https://github.com/microsoft/azure-pipelines-tasks/blob/8d49d17fc0b84ae880b5ad2c7e18936d3ff0ca2e/docs/taskversionbumping.md#bumping-task-version), the minor version should be equivalent to the[ sprint version  ](https://whatsprintis.it/)

Reply:
interesting, does this versioning requirement introduce the possibility of doing a release without bumping the minor version?

Reply:
Yes if the minor version and sprint number are the same will increase the patch number